systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -193,18 +196,20 @@ public void testKRaftMode(ExtensionContext extensionContext) {
     void testKafkaNodePoolFeatureGate(ExtensionContext extensionContext) {
         assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
 
-        final TestStorage testStorage = new TestStorage(extensionContext, CO_NAMESPACE);
+        final TestStorage testStorage = new TestStorage(extensionContext);
 
         List<EnvVar> coEnvVars = new ArrayList<>();
         coEnvVars.add(new EnvVar(Environment.STRIMZI_FEATURE_GATES_ENV, "+KafkaNodePools", null));
         
         clusterOperator = this.clusterOperator.defaultInstallation(extensionContext)
             .withExtraEnvVars(coEnvVars)
+            .withNamespace(Constants.CO_NAMESPACE)
```

Comment:
I think you do not need explicitly use `CO_NAMESPACE` here. It is encapsulated in `defaultInstallation` method I assume.

Reply:
good catch thanks